lib/toml/keyvalue.rb

```diff
@@ -2,21 +2,43 @@ module TOML
   class ValueOverwriteError < StandardError; end
 
   class Keyvalue
-    attr_reader :key, :value
+    attr_reader :value, :symbolize_keys
 
     def initialize(key, value)
-      @key, @value = key, value
+      @key, @value, symbolize_keys = key, value, false
```

Comment:
Useless assignment to variable - `symbolize_keys`. Use `_` or `_symbolize_keys` as a variable name to indicate that it won't be used.